fundamentals/fundamentals-bio/src/test/scala/izumi/fundamentals/bio/test/SchedulerTest.scala

```diff
@@ -0,0 +1,157 @@
+package izumi.fundamentals.bio.test
+
+import izumi.functional.bio.impl.SchedulerZio
+import izumi.functional.bio.retry.RetryPolicy.{ControllerDecision, RetryFunction}
+import izumi.functional.bio.retry.{RetryPolicy, Scheduler2, toZonedDateTime}
+import izumi.functional.bio.{F, Monad2, Primitives2, UnsafeRun2}
+import org.scalatest.wordspec.AnyWordSpec
+import zio.internal.Platform
+
+import java.time.ZonedDateTime
+import java.time.temporal.ChronoUnit
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration._
+
+class SchedulerTest extends AnyWordSpec {
+  //private val monixRunner = UnsafeRun2.createMonixBIO(TestScheduler(), bio.IO.defaultOptions)
+  private val zioRunner = UnsafeRun2.createZIO(Platform.default)
+
+  // TODO: monix requires timer instance which apparently I should make by myself since I cannot instantiate a default one?
+  //private val monixScheduler = new SchedulerMonix(cats.effect.Timer)
+  private val zioTestClock = zio.Has(zio.clock.Clock.Service.live)
```

Comment:
@neko-kai, I haven't found other way to test Scheduler except to use runner to run all effects.
I have not problem with ZIO instances, but I am kinda struggling with Monix, since I cannot access a Timer that is used in IOApp (default one). I guess the only way is to write one by myself (which is actually not a problem at all), I am just wondering, maybe I missed something?

Reply:
@VladPodilnyk You may use `Scheduler.global`

Reply:
@neko-kai, thanks, I've summoned it implicitly. For some reason I thought compiler wouldn't find it...